docker/Dockerfile

```diff
@@ -51,16 +54,17 @@ RUN apt-get update && \
 # deleting default nginx conf
 RUN rm /etc/nginx/sites-enabled/default && rm /etc/nginx/sites-available/default
 
-# /ssl includes certs and acme.sh config - mount /ssl as a volume for persistency
-RUN mkdir -p /ssl/certs && \
-    mkdir -p /ssl/acme.sh/config && mkdir -p /ssl/acme.sh/certs && \
-    chown -R www-data:www-data /ssl/certs
+
+# /ssl includes certs and acme.sh config - mount $PERISTENT_DATA_PATH as a volume for persistency
```

Comment:
Update comment with new env

Reply:
@anonym-HPI Unfortunately, you've missed `DFM_` in the env name twice in this line.

Reply:
@anonym-HPI Unfortunately, now you've lost the `$`.